src/routes/index.ts

```diff
@@ -8,48 +9,106 @@ import {
   renameFilesToChecksum,
 } from 'middlewares/files'
 import path from 'path'
+import qs from 'querystring'
+import redis from 'redis'
 import logger from 'utils/logger'
 
 const router = express.Router()
 
+const redisClient = redis.createClient({ url: process.env.REDIS_URI })
+
+const DEFAULT_EXPIRE = Number(process.env.CACHE_EXPIRE || 60)
+
+const cache = ExpressRedisCache({
+  client: redisClient,
+  prefix: 'file',
+  expire: DEFAULT_EXPIRE, // 1 min,
+})
+
+cache.on('message', message => logger.info('Cached %s', message))
+cache.on('connected', () => logger.info('Cache redis server connected'))
+cache.on('disconnected', () => logger.info('Cache redis server connected'))
+cache.on('error', error => logger.info('Cache redis server error %o', error))
```

Comment:
this should be `logger.error`
```suggestion
cache.on('error', error => logger.error('Cache redis server error %o', error))
```